spring-cloud-function-samples/function-sample-azure/pom.xml

```diff
@@ -40,7 +40,7 @@
 		<dependency>
 			<groupId>com.microsoft.azure</groupId>
 			<artifactId>azure-functions-java-core</artifactId>
-			<version>1.0.0-beta-2</version>
+			<version>1.0.0-beta-3</version>
```

Comment:
Probably this ought to go in the adapter itself right? There is no need for samples (user code) to know this version id?

Reply:
Good point.